source/speakers/index.html.erb

```diff
@@ -9,6 +9,39 @@ created_by:
   name: Mike Rogers
 ---
 
+<div class="speaker__list">
+
+  <% all_speakers.each do |speaker| %>
+    <div class="speaker">
+      <div class="speaker__avatar">
+        <%= image_tag speaker.avatar_url, alt: "#{speaker.name} avatar" %>
+      </div>
+
+      <div class="speaker__details">
+        <h2 class="speaker__name"><%= speaker.name %></h2>
+        <div class="speaker__socials">
+          <% link_to "https://twitter.com/#{speaker.twitter_username}" do %>
+            <%= image_tag 'twitter.svg', width: 24, height: 24 %>
+          <% end %>
+
+          <% link_to "https://github.com/#{speaker.github_username}" do %>
+            <%= image_tag 'github.svg', width: 24, height: 24 %>
+          <% end %>
+        </div>
+
+        <p><%= speaker.bio %></p>
+
+        <p>Talks: <%= speaker.talks.count %></p>
+
+        <p>
+          <%= link_to speaker.blog, speaker.blog %>
+        </p>
+      </div>
```

Comment:
I know we've taken inspiration from some other website on this, but I'm pretty sure we'll never get most of these details, so lets pare it right back.  I quite like the svg icons for the twitter / GitHub links, but I also know that without a huge amount of effort we're probably not going to get twitter _and_ GitHub links for most of our speakers, and for some we don't even have those, some folk just link to LinkedIn and personal websites were the preferred link back in the past.  Perhaps we can just introspect on the one link we do have and render an appropriate icon?

Reply:
> Perhaps we can just introspect on the one link we do have and render an appropriate icon?

I agree :) I was hoping to find a way to pull in a bit more information, I think I'll save that for another PR :)